WalletWasabi.Tests/UnitTests/WabiSabi/Backend/PostRequests/RemoveInputTests.cs

```diff
@@ -32,7 +33,7 @@ public async Task SuccessAsync()
 			await using ArenaRequestHandler handler = new(cfg, new Prison(), arena, new MockRpcClient());
 
 			// There's no such alice yet, so success.
-			var req = new InputsRemovalRequest(round.Id, Guid.NewGuid());
+			var req = new InputsRemovalRequest(round.Id, new uint256(123456));
```

Comment:
Why can't we have Zero here?

Reply:
We can have Zero. I just didn't use it because we weren't using `Guid.Empty` but `Guid.NewGuid()` however using `uint256.Zero` is okay too. Do you think is better to use something less noisy?

Reply:
I changed this hardcoded value by `BitcoinFactory.CreateUInt256()`.